### PR TITLE
Update Data Browser html file

### PR DESCRIPTION
**Pull request by dmitrizagidulin**
To use the new auth client.

### CODE REVIEW

File: config/defaults.js [hunk 1]
```diff
@@ -10,5 +10,6 @@ module.exports = {
   'dbPath': './.db',
   'port': 8443,
   'serverUri': 'https://localhost:8443',
-  'webid': true
+  'webid': true,
+  'dataBrowserPath': 'default'
```
Comment (RubenVerborgh) [inline config/defaults.js]: What are possible alternative values? Why isn't this `static/databrowser.html`?

**Comment by RubenVerborgh** [comment]
Any steps to test?

File: config/defaults.js (hunk 1 above)
Comment (dmitrizagidulin) [reply config/defaults.js]: Not quite sure why it was done this way. I do know that it's what activates it.

**Comment by dmitrizagidulin** [comment]
To test, start up the server and request some rdf file on the server, say /settings/prefs.ttl or something. (Test on a browser that doesn't have an rdf plugin)

**Comment by RubenVerborgh** [comment]
I get `Uncaught ReferenceError: Mashlib is not defined`.

**Comment by dmitrizagidulin** [comment]
Ah damn, I think I forgot a git push

**Comment by dmitrizagidulin** [comment]
Maybe there's a delay in propagation to github.io..

**Comment by RubenVerborgh** [comment]
No change unfortunately.

**Comment by dmitrizagidulin** [comment]
Found the issue - mashlib repo was still set to serve gh-pages instead of master on github.io. Should be fixed shortly..

**Comment by dmitrizagidulin** [comment]
(might require a shift-refresh)

**Comment by RubenVerborgh** [comment]
(BTW regarding gh-pages and master, see https://github.com/linkeddata/rdflib.js/pull/201.)

**Pull request merged**